docs/snippets/azure/AppHost/Program.ConfigureCosmosInfra.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Azure.Provisioning.CosmosDB;
+
+internal static partial class Program
+{
+    public static void ConfigureCosmosInfra(IDistributedApplicationBuilder builder)
+    {
+        // <configure>
+        builder.AddAzureCosmosDB("cosmos-db")
+            .ConfigureInfrastructure(infra =>
+            {
+                var cosmosDbAccount = infra.GetProvisionableResources()
+                                           .OfType<CosmosDBAccount>()
+                                           .Single();
+
+                cosmosDbAccount.Kind = CosmosDBAccountKind.MongoDB;
```

Comment:
Does this work when you use the client integration?

Reply:
I'm not certain, but why wouldn't it?

Reply:
I don't know I've never tried it with out client integration. Does it just work like magic?